src/foam/nanos/auth/Region.js

```diff
@@ -43,13 +50,13 @@ foam.CLASS({
       name: 'getRegionCode',
       type: 'String',
       code: function() {
-        if ( this.code && this.code.length > 3 ) {
-          return this.code.substring(3, this.code.length);
+        if ( this.isoCode && this.isoCode.length > 3 ) {
+          return this.isoCode.substring(3, this.isoCode.length);
         }
         return '';
```

Comment:
you need to return isoCode here, no?

Reply:
this returns the region code section of the isoCode. so for "DE-BE" it returns "BE".